pkg/health/nats_stream.go

```diff
@@ -0,0 +1,22 @@
+package health
+
+import (
+	"context"
+
+	"github.com/nats-io/nats.go"
+	"github.com/rancher/opni/pkg/alerting/shared"
+	natsutil "github.com/rancher/opni/pkg/util/nats"
+)
+
```

Comment:
See other comment, move this to pkg/alerting

Reply:
This is intended to be used for logging / topology / alerting

Reply:
Done